pom.xml

```diff
@@ -73,13 +72,15 @@
     <version.org.gwtbootstrap3>0.9.4</version.org.gwtbootstrap3>
     <version.org.jboss.errai>4.3.3.Final</version.org.jboss.errai>
     <version.org.optaplanner>${project.version}</version.org.optaplanner>
+    <version.org.kie.soup>${project.version}</version.org.kie.soup>
     <version.org.slf4j>1.7.25</version.org.slf4j>
     <version.org.webjars.bower.org.patternfly>3.18.1</version.org.webjars.bower.org.patternfly>
     <version.org.webjars.Eonasdan-bootstrap-datetimepicker>4.17.47</version.org.webjars.Eonasdan-bootstrap-datetimepicker>
     <version.org.webjars.momentjs>2.22.2</version.org.webjars.momentjs>
     <version.org.wildfly-plugins.wildfly-maven-plugin>1.2.0.Final</version.org.wildfly-plugins.wildfly-maven-plugin>
     <version.org.wildfly>14.0.1.Final</version.org.wildfly>
     <version.org.codehaus.cargo.cargo-maven2-plugin>1.6.11</version.org.codehaus.cargo.cargo-maven2-plugin>
+    <version.org.mockito>2.12.0</version.org.mockito>
```

Comment:
You introduced this property but I don't see it used anywhere? Also, doesn't kie-parent define the version of Mockito?

Reply:
it is overriding the kie-parent version property. Here you used 2.12.0, but kie-parent uses 1.x and you used not backward compatible API/method, so therefore there is required that overriding

Reply:
or the version property is defined in IP BOM ;)

Reply:
Does simply changing the version String actually do that? I would think you need to put something in dependency management in order to actually do that. If that the case, can you please add a comment above the property explaining that we are overriding kie-parent version of mockito since we are using gwtmockito (which, if I remember correctly, is the dependency that set the mockito version)?